api-reference/10 UI Components/dxScrollable/3 Methods/scrollBy(distance).md

```diff
@@ -6,7 +6,7 @@ id: dxScrollable.scrollBy(distance)
 Scrolls the content by a specific distance.
 
 ##### param(distance): Number | Object
-The distance in pixels.
+The distance in pixels. The object has the following structure: { left: value1, top: value2 }.
```

Comment:
It wasn't clear to me what object you are referring to. It would be better to elaborate on this a bit:

```suggestion
The distance in pixels.     
When you specify this parameter with an object, it should have the following structure: `{ left: value1, top: value2 }`.
```